go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/stackrox/scanner
 
-go 1.21
+go 1.21.8
```

Comment:
Is this change required?

Reply:
This is updated by `go mod tidy`. I'm not sure about the reasons, it could be because of `stackrox/stackrox` - https://github.com/stackrox/stackrox/blob/195784f810aea44b077f9bda82851c91eb84b45f/go.mod#L3

Reply:
makes sense